corehq/apps/hqwebapp/utils/bootstrap/paths.py

```diff
@@ -13,13 +13,40 @@
     "ex-submodules": COREHQ_BASE_DIR / "ex-submodules",
 }
 GRUNTFILE_PATH = COREHQ_BASE_DIR.parent / "Gruntfile.js"
+IGNORED_PATHS_BY_APP = {
+    "hqwebapp": [
+        "hqwebapp/js/resource_versions.js",
+        "hqwebapp/base.html",
+        "hqwebapp/base_mobile.html",
+        "hqwebapp/crispy/",
+        "hqwebapp/includes/inactivity_modal_data.html",
+        "hqwebapp/includes/core_libraries.html",
+        "hqwebapp/includes/ui_element_js.html",
+        "hqwebapp/partials/requirejs.html",
+        # todo, remove these files and fully split:
+        "hqwebapp/iframe_domain_login.html",
+        "hqwebapp/bulk_upload.html",
+        "hqwebapp/crispy/single_crispy_form.html",
+        "hqwebapp/spec/mocha.html",
+        "hqwebapp/js/maintenance_alerts.js",
+        "hqwebapp/spec/widgets_spec.js",
+    ]
+}
 
 
 def is_split_path(path):
     path = str(path)
     return "/bootstrap3/" in path or "/bootstrap5/" in path
 
 
+def is_ignored_path(app_name, path):
+    path = str(path)
+    for ignored_path in IGNORED_PATHS_BY_APP.get(app_name, []):
+        if ignored_path in path:
```

Comment:
For `hqwebapp` it probably doesn't matter, but I can imagine `ignored_path` values that may be more generic, and could match multiple sets of paths within the repository. Example: `api/...` would match `corehq/apps/api/...` and `corehq/apps/formplayer_api/...`. What do you think of anchoring all paths to the root of the repository (which implies using a different matching technique from `sub_path in full_path`)?

Reply:
i will optimize for that if and when it's needed. right now it's really not necessary